client/src/django.ts

```diff
@@ -97,8 +101,15 @@ const djangoClient = {
     const { data } = await apiClient.get(`/scans/${scanId}`);
     return data;
   },
-  async setDecision(scanId: string, decision: string, comment: string) {
-    const { data } = await apiClient.post('/scan-decisions', { scan: scanId, decision, note: comment });
+  async setDecision(
+    scanId: string,
+    decision: string,
+    comment: string,
+    userIdentifiedArtifacts: Object,
```

Comment:
```suggestion
    userIdentifiedArtifacts: { present: Artifact[], absent: Artifact[] },
```
Artifacts are thrown around a lot, I think they'd benefit from having a well defined type.

Reply:
The objects passed around are actually just strings, and I don't think they need more information, but we can absolutely replace `Object` here with `{ present: String[], absent: String[] }`.